app/subjects/comment-form.cjsx

```diff
@@ -78,23 +74,25 @@ module.exports = React.createClass
 
   render: ->
     return <Loading /> if @state.loading
-    return @notSetup() unless @state.boards
+    return @noDefaultBoardMessage() unless @state.boards
     return @loginPrompt() unless @props.user
 
-    <div>
-      <div className="tabbed-content">
-        <div className="tabbed-content-tabs">
-          <div className="subject-page-tabs">
-            <div className="tabbed-content-tab #{if @state.tab is 0 then 'active' else ''}" onClick={=> @setState({tab: 0})}>
-              Add a note about this subject
-            </div>
-
-            <div className="tabbed-content-tab #{if @state.tab is 1 then 'active' else ''}" onClick={=> @setState({tab: 1})}>
-              Start a new discussion
+    if @state.boards.length < 1
+      @renderTab()
+    else
+      <div>
+        <div className="tabbed-content">
+          <div className="tabbed-content-tabs">
+            <div className="subject-page-tabs">
+              <button type="button" className="create-discussion-or-comment #{if @state.tab is 0 then 'active' else ''}" onClick={=> @setState({tab: 0})}>
```

Comment:
I don't think the suggestion to use button markup here meant that it should also be styled as a button. I think the tab styling should remain. The buttons no longer get correct active styling anymore either. 

<img width="966" alt="screen shot 2017-10-02 at 2 49 25 pm" src="https://user-images.githubusercontent.com/5016731/31096199-f883540a-a780-11e7-8dd6-fe9d2ba275d1.png">

How the tabs looked:

<img width="967" alt="screen shot 2017-10-02 at 2 52 02 pm" src="https://user-images.githubusercontent.com/5016731/31096309-4b9996b8-a781-11e7-8013-02f66bafa577.png">